assets/in

```diff
@@ -63,7 +63,11 @@ git clone --single-branch $depthflag $uri $branchflag $destination
 cd $destination
 
 git fetch origin refs/notes/*:refs/notes/*
-git checkout -q $ref
+until `git checkout -q $ref`; do
```

Comment:
This will result in an infinite loop if `$ref` is no longer available anywhere (i.e. someone `push -f`ed).

Reply:
Hum. Good catch!